w1: [[[[-1.6650e-01, -1.6682e-01, -1.6663e-01],
       [-7.5112e-06, -4.0844e-05,  3.7997e-05],
       [ 1.6670e-01,  1.6667e-01,  1.6660e-01]]],
     [[[-1.6653e-01,  3.0289e-05,  1.6651e-01],
       [-1.6677e-01, -5.4666e-05,  1.6669e-01],
       [-1.6665e-01,  1.2168e-04,  1.6665e-01]]],
     [[[-1.3175e-04,  1.2513e-01,  2.4983e-01],
       [-1.2505e-01,  3.6682e-06,  1.2503e-01],
       [-2.4988e-01, -1.2494e-01,  9.7242e-06]]],
     [[[ 2.4968e-01,  1.2499e-01, -1.5196e-05],
       [ 1.2511e-01, -2.3487e-04, -1.2501e-01],
       [ 2.1485e-04, -1.2496e-01, -2.4978e-01]]]]
w2: [[[[-1.6604e-01, -1.6719e-01, -1.6669e-01],
       [ 4.1097e-04, -7.9650e-05,  9.1703e-05],
       [ 1.6636e-01,  1.6711e-01,  1.6602e-01]]],
     [[[-1.6644e-01,  1.6302e-04,  1.6659e-01],
       [-1.6714e-01,  3.3449e-05,  1.6717e-01],
       [-1.6642e-01,  1.3836e-04,  1.6591e-01]]],
     [[[-2.8285e-04,  1.2527e-01,  2.4939e-01],
       [-1.2541e-01,  1.6673e-05,  1.2501e-01],
       [-2.4916e-01, -1.2515e-01,  3.0937e-04]]],
     [[[ 2.4936e-01,  1.2532e-01,  5.7826e-05],
       [ 1.2510e-01, -7.0027e-05, -1.2552e-01],
       [ 1.5795e-04, -1.2541e-01, -2.4900e-01]]]]
w3: [[[[-1.6498e-01, -1.6708e-01, -1.6560e-01],
       [ 9.8741e-04,  6.1481e-04, -2.3462e-03],
       [ 1.6511e-01,  1.6745e-01,  1.6584e-01]]],
     [[[-1.6636e-01,  5.5259e-04,  1.6628e-01],
       [-1.6795e-01,  8.4194e-04,  1.6589e-01],
       [-1.6569e-01,  3.2529e-04,  1.6612e-01]]],
     [[[ 3.5657e-04,  1.2612e-01,  2.4948e-01],
       [-1.2560e-01,  3.0604e-04,  1.2374e-01],
       [-2.4845e-01, -1.2534e-01, -6.1795e-04]]],
     [[[ 2.4942e-01,  1.2524e-01, -1.5642e-04],
       [ 1.2534e-01, -3.7394e-04, -1.2408e-01],
       [-3.9571e-04, -1.2554e-01, -2.4946e-01]]]]
w_out: [[[[1.2696]],
         [[1.4777]],
         [[1.7409]],
         [[1.7916]],
         [[0.9110]],
         [[0.8669]],
         [[1.3450]],
         [[1.3525]],
         [[0.2810]],
         [[0.0376]],
         [[0.5175]],
         [[0.5921]]]]
